04-overall-workflow.Rmd

```diff
@@ -89,7 +91,7 @@ Repositories usually have either a "main" or a "master" branch. The two terms re
 ::::
 
 - When you've made enough changes to complete your work item checkout the main 
-branch in your local repository do a pull in case anyone else had made changes.
+branch in your local repository and pull in case anyone else had made changes.
```

Comment:
Would it be any clearer for a new user to explicitly say "git pull" rather than just "pull" here?